sorc/chgres_cube.fd/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+set(fortran_src
+    atmosphere.F90
+    chgres.F90
+    grib2_util.F90
+    input_data.F90
+    model_grid.F90
+    program_setup.f90
+    search_util.f90
+    static_data.F90
+    surface.F90
+    utils.f90
+    write_data.F90)
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "-g -traceback -r8 -i4 -convert big_endian -assume byterecl")
```

Comment:
I think ``-i4`` flag is redundant. By default integers are 4-byte wide.

Reply:
I removed all -i4 flags.